modular_skyrat/modules/loadouts/loadout_items/loadout_datum_glasses.dm

```diff
@@ -174,6 +182,82 @@ GLOBAL_LIST_INIT(loadout_glasses, generate_loadout_items(/datum/loadout_item/gla
 	item_path = /obj/item/clothing/glasses/hud/diagnostic/prescription
 	restricted_roles = list(JOB_RESEARCH_DIRECTOR,JOB_SCIENTIST, JOB_ROBOTICIST)
 
+/datum/loadout_item/glasses/aviator_security
+	name = "Security HUD Aviators Sunglasses"
```

Comment:
```suggestion
	name = "Security HUD Aviators"
```
Just "aviators" suffices here and avoids the name getting too lengthy. Apply to the rest of the loadout datums